kubernetes/package-kafka/kafkaprovider.yml

```diff
@@ -0,0 +1,56 @@
+---
+apiVersion: v1
+kind: Service
```

Comment:
Do we need a service?  I didn't see any references to it, but maybe I missed something.